atmos_dyamond_summer/create_artifact.jl

```diff
@@ -1,8 +1,276 @@
 using ClimaArtifactsHelper
+using Interpolations
+using NCDatasets
+using DataStructures
+import ClimaParams
+import Thermodynamics as TD
+import Thermodynamics.Parameters as TP
 
 const FILE_URL = "https://swift.dkrz.de/v1/dkrz_ab6243f85fe24767bb1508712d1eb504/SAPPHIRE/DYAMOND/ifs_oper_T1279_2016080100.nc"
 const FILE_PATH = "ifs_oper_T1279_2016080100.nc"
 
 artifact_name = "DYAMOND_summer_initial_conditions"
 
 create_artifact_guided_one_file(FILE_PATH; artifact_name = artifact_name, file_url = FILE_URL)
+
+const FT = Float32
+const H_EARTH = 7000.0
+const P0 = 1e5
+const log_P0 = log(P0)
+const z_min, z_max = 10.0, 80E3
+
+Plvl(z) = P0 * exp(-z / H_EARTH)
+Plvl_inv(P) = -H_EARTH * log(P / P0)
+Plvl_inv2(log_p_by_p0) = -H_EARTH * log_p_by_p0
+
+compute_P(hyam, hybm, surfacepress) = hyam + hybm * surfacepress
+
+param_set = TP.ThermodynamicsParameters(FT)
+const grav = TP.grav(param_set)
+
+function interpz_3d(target_z, z3d, var3d)
+    nx, ny, nz = size(z3d)
+    target_var3d = similar(var3d, nx, ny, length(target_z))
+    @inbounds begin
+        for yi in 1:ny, xi in 1:nx
+            source_z = view(z3d, xi, yi, :)
+            itp = extrapolate(
+                interpolate(
+                    (source_z,),
+                    (view(var3d, xi, yi, :)),
+                    Gridded(Linear()),
+                ),
+                Flat(),
+            )
+            target_var3d[xi, yi, :] .= itp.(target_z)
+        end
+    end
+
+    return target_var3d
+end
+
+function create_initial_conditions(infile, outfile; skip = 1)
+    ncin = NCDataset(infile, "r")
+    ncout = NCDataset(outfile, "c")
```

Comment:
Could you copy over the `history` attribute and add a note that we modified this file? (See my PR)